py3status/modules/file_status.py

```diff
@@ -68,12 +65,18 @@ def file_status(self):
                 'cached_until': self.py3.CACHE_FOREVER,
             }
 
-        if exists(self.path):
-            icon = self.icon_available
-            color = self.py3.COLOR_GOOD
-        else:
-            icon = self.icon_unavailable
-            color = self.py3.COLOR_BAD
+        paths = glob(expanduser(self.path))
+        if isinstance(paths, str):
+            paths = [paths]
+
+        icon = self.icon_unavailable
+        color = self.py3.COLOR_BAD
+
+        for path in paths:
+            if exists(path):
+                icon = self.icon_available
+                color = self.py3.COLOR_GOOD
+                break
 
         response = {
             'cached_until': self.py3.time_in(self.cache_timeout),
```

Comment:
`cache_timeout = -1` in your example means once. You may want `cache_timeout = 0` instead.

Reply:
In my example I use `py3-cmd refresh file_status` for try ;)